std/typecons.d

```diff
@@ -4684,16 +4684,24 @@ if (Targets.length >= 1 && allSatisfy!(isMutable, Targets))
             alias type = F;
         }
 
+        // Remove NVI members
+        template RemNVI(members...)
```

Comment:
I don't think there's a need for such an abbreviation here. Please change this to something more descriptive like `removeNonVirtual`.

Reply:
Really ? can someone confirm, I'm afraid that `RemoveNonVirtualMembers` is a bit too much for a local helper template, also NVI is quite known. If you look at the comments inside the parent template there's one that contains NVI too.